### DISCUSSION TITLE
Network error: JSON.parse

### DISCUSSION THREAD

**Discussion by veg4c0re** [requarks/wiki#5325] [Error / Bug Report] [answered]
Hi,

for some reason I get the Error: `Network error: JSON.parse: unexpected character at line 1 column 1 of the JSON data`. This happens, for example, when saving a page or applying changes at my backup settings. I had to change the backup settings in the DB directly.

**Comment by veg4c0re** [comment, marked as answer]
I fixed the error by changing the `nginx.conf`

```
        client_body_buffer_size 1k;
        client_header_buffer_size 1k;
        client_max_body_size 1k;
        large_client_header_buffers 2 1k;
```

This buffer overflow prevention should be increased or comment it completlly out.

**Reply by fo-bo** [reply]
> I fixed the error by changing the `nginx.conf`
> 
> ```
>         client_body_buffer_size 1k;
>         client_header_buffer_size 1k;
>         client_max_body_size 1k;
>         large_client_header_buffers 2 1k;
> ```
> 
> This buffer overflow prevention should be increased or comment it completlly out.

how could i fix this if i run wiki.js in docker&

**Reply by veg4c0re** [reply]
@fo-bo there must be something equivalent to `client_max_body_size 1k;` in your docker-compose.yml.
NGINX has a default upload size. Same must be for docker which you have to override.
